site/content/en/docs/developer-guide/plugin_integration_test_framework.md

```diff
@@ -7,7 +7,8 @@ title: Plugin Integration Test Framework
 Assumed you are at the `./plugins`:
 
 1. Run `make build-test-so` to build the Go plugins.
-2. Run `go test -v ./tests/integration -run TestPluginXX` to run the selected tests.
+2. Place the built `libgolang.so` into the `./tests/integration` directory.
```

Comment:
Interesting. The `make build-test-so` should already place `libgolang.so` into the `./tests/integration` directory:
https://github.com/mosn/htnn/blob/de1d13b57a48881edf83c1a03d62a2d38a1b821f/plugins/Makefile#L33

Why doesn't it work?